python/tests/integration/completions/test_utils.py

```diff
@@ -19,3 +22,23 @@ async def retry(func, retries=20):
                 raise
             await asyncio.sleep(max(min(i, max_delay), min_delay))
     return None
+
+
+def setup_onnx_gen_ai_chat_completion():
+    prompt = """{% for message in messages %}
+                    {% if message['content'] is not string %}
+                        {{'<|image_1|>'}}
+                    {% else %}
+                        {% if message['role'] == 'system' %}
+                            {{'<|system|>\n' + message['content'] + '<|end|>\n'}}
+                        {% elif message['role'] == 'user' %}
+                            {{'<|user|>\n' + message['content'] + '<|end|>\n'}}
+                        {% elif message['role'] == 'assistant' %}
+                            {{'<|assistant|>\n' + message['content'] + '<|end|>\n' }}
+                        {% endif %}
+                    {% endif %}
+                {% endfor %}
+                <|assistant|>"""
+
+    prompt_template_config = PromptTemplateConfig(template=prompt, template_format="jinja2")
+    return OnnxGenAIChatCompletion(prompt_template_config)
```

Comment:
Is this going to always fail because we don't have a Onnx setup in our test pipeline?

Reply:
Currently it would skip the tests, since the `ONNX_GEN_AI_MODEL_PATH` Env Variable is not set because the method is called in https://github.com/nmoeller/semantic-kernel/blob/af15de633a321001bdce67e8a569aed4879de500/python/tests/integration/completions/chat_completion_test_base.py#L98.

Then the skip variable is set, i also tested this locally to validate this.